website/docs/usage/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+slug: /usage
+sidebar_position: 2
+title: "Usage"
+---
+import versions from '@site/versions';
+import CodeBlock from '@theme/CodeBlock';
+
+# SQLiter
+
+## Usage
+
+### Open a connection (file based DB)
+
+```kotlin
+// open a very basic connection
+val config = DatabaseConfiguration("my-database.db", 1, { db -> ... })
+val manager = createDatabaseManager(config)
+val connection = manager.createMultiThreadedConnection()
+
+// execute sql queries
+
+// don't forget to close the connection
+connection.close()
+```
+
+### Open a connection (in-memory DB)
+
+:::info Why do in-memory databases need names?
+
+Sqlite lets you create an in memory db with multiple connections using a shared memory cache with the supplied
```

Comment:
This we should update. *if* you pass in a name you need to manage the connections. If you pass in null, `:memory:` should just auto-close when you disconnect.